contracts/tgrade-validator-voting/src/contract.rs

```diff
@@ -56,8 +56,18 @@ pub fn execute(
         } => {
             // Migrate contract needs confirming that sender (proposing member) is an admin
             // of target contract
-            if let ValidatorProposal::MigrateContract { ref contract, .. } = proposal {
+            if let ValidatorProposal::MigrateContract {
+                ref contract,
+                ref migrate_msg,
+                ..
+            } = proposal
+            {
                 confirm_admin_in_contract(deps.as_ref(), &env, contract.clone())?;
+                dbg!(migrate_msg);
```

Comment:
I would remove both these `dbg!` statements before merging. But the check looks good.

wondering about pulling this all into some `validate` method on `ValidatorProposal` but not strictly necessary yet.

Reply:
Missed that one.